### PR TITLE
adding properties section in readme

### PR DESCRIPTION
**Pull request by Keyshal-211**
## Description of changes
Please provide a short description of the change or if the change is sufficiently described in the GitHub issue, provide a link to the issue.

  
## How has this been tested?
Please describe the test strategy you followed.

- [ ] automated unit test
- [ ] automated integration test
- [ ] automated acceptance test
- [ ] manual, exploratory test

In case of manual test, please document the test well including a set of user instructions and prerequisites. Each including an action, it's result, and where appropriate a screenshot.
In case of automated tests, the test coverage should be reasonably high, especially on Service classes and other classes with complex logic.
    
## Checklist before requesting a review
- [X] My code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My code fulfills all acceptance criteria
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] The test coverage (line and branch) is reasonably high, especially on Service classes and other classes with complex logic
- [ ] I have made corresponding changes to the documentation
- [ ] I have added explanation of architectural decision and rationales to [wiki/adr](https://github.com/IT-REX-Platform/wiki/tree/main/adr)
- [ ] I have updated the changes in the ticket description

## Checklist for reviewer
- [ ] The code works and does not throw errors
- [ ] The code is easy to understand and there are no confusing parts
- [ ] The code follows the [coding guidelines](https://github.com/IT-REX-Platform/wiki/blob/main/dev-manuals/backend/coding-guidelines.md) of this project
- [ ] The code change accomplishes what it is supposed to do
- [ ] I cannot think of any use case in which the code does not behave as intended
- [ ] The added and existing tests reasonably cover the code change
- [ ] I cannot think of any test cases, input or edge cases that should be tested in addition
- [ ] Description of the change is included in the documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,30 @@
...
+| USER_SERVICE_URL       | URL for user service GraphQL       | http://host.docker.internal:5001/graphql                                   | http://localhost:3500/v1.0/invoke/user-service/method/graphql       |
+| FLASHCARD_SERVICE_URL  | URL for flashcard service GraphQL  | http://host.docker.internal:6001/graphql                                   | http://localhost:3500/v1.0/invoke/flashcard-service/method/graphql  |
+| REWARD_SERVICE_URL     | URL for reward service GraphQL     | http://host.docker.internal:7001/graphql                                   | http://localhost:3500/v1.0/invoke/reward-service/method/graphql     |
+| SKILLLEVEL_SERVICE_URL | URL for skilllevel service GraphQL | http://host.docker.internal:8001/graphql                                   | http://localhost:3500/v1.0/invoke/skilllevel-service/method/graphql |
+| QUIZ_SERVICE_URL       | URL for quiz service GraphQL       | http://host.docker.internal:9001/graphql                                   | http://localhost:3500/v1.0/invoke/quiz-service/method/graphql       |                                                               |
```
Comment (PaulBredl) [inline README.md:24]: all properties are relevant for deployment

File: README.md [hunk 2]
```diff
@@ -4,16 +4,11 @@ A gateway which uses graphql mesh to split a query into smaller queries to the d
...
-|------------------|-------------------------|--------------------------|---------------------------|
-| GATEWAY_PORT     | PostgreSQL database URL | 0.0.0.0                  | 0.0.0.0                   |
-| GATEWAY_HOSTNAME | Database username       | 8080                     | 8080                      |
-| DAPR_HTTP_PORT   | Dapr HTTP Port          | -                        | 3500                      |
-
-
-### Other properties
 | Name                   | Description                        | Value in Dev Environment                                                   | Value in Prod Environment                                           |
 |------------------------|------------------------------------|----------------------------------------------------------------------------|---------------------------------------------------------------------|
+| GATEWAY_PORT           | PostgreSQL database URL            | 0.0.0.0                                                                    | 0.0.0.0                                                             |
```
Comment (PaulBredl) [inline README.md:9]: Sorry I just saw. These description are completely wrong for the first two. Also they are mixed up. The port has a host as value and the host has a port as value??

**Review by PaulBredl — CHANGES_REQUESTED** [review]
Some descriptions are wrong

File: README.md [hunk 3]
```diff
@@ -1,5 +1,29 @@
...
+| CONTENT_SERVICE_URL    | URL for content service GraphQL    | http://host.docker.internal:4001/graphql | http://localhost:3500/v1.0/invoke/content-service/method/graphql    |
+| USER_SERVICE_URL       | URL for user service GraphQL       | http://host.docker.internal:5001/graphql | http://localhost:3500/v1.0/invoke/user-service/method/graphql       |
+| FLASHCARD_SERVICE_URL  | URL for flashcard service GraphQL  | http://host.docker.internal:6001/graphql | http://localhost:3500/v1.0/invoke/flashcard-service/method/graphql  |
+| REWARD_SERVICE_URL     | URL for reward service GraphQL     | http://host.docker.internal:7001/graphql | http://localhost:3500/v1.0/invoke/reward-service/method/graphql     |
+| SKILLLEVEL_SERVICE_URL | URL for skilllevel service GraphQL | http://host.docker.internal:8001/graphql | http://localhost:3500/v1.0/invoke/skilllevel-service/method/graphql |
+| QUIZ_SERVICE_URL       | URL for quiz service GraphQL       | http://host.docker.internal:9001/graphql | http://localhost:3500/v1.0/invoke/quiz-service/method/graphql       |                                                               |
+
+
+### Other properties
```
Comment (PaulBredl) [inline README.md:22]: Sorry if it wasn't clear, but JWKS_URL is also relevant for prod

**Review by PaulBredl — CHANGES_REQUESTED** [review]
See comment

**Review by PaulBredl — APPROVED** [review]
